### PR TITLE
#0: Added option to filter perf results to within start/stop signposts

### PR DESCRIPTION
**Pull request by AleksKnezevic**

### CODE REVIEW

**Review by arakhmati — APPROVED** [review]

**Review by mo-tenstorrent — APPROVED** [review]

**Review by tt-rkim — APPROVED** [review]

File: pytest.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [pytest]
-timeout = 2400
+timeout = 102400
```
Comment (TT-billteng) [inline pytest.ini:2]: Is this an accident?

**Review by TT-billteng — CHANGES_REQUESTED** [review]
pytest.ini

Also, can you find a way to not use `SLOW_MATMULS` ?

File: pytest.ini (hunk 1 above)
Comment (AleksKnezevic) [reply pytest.ini:2]: Yes, it is.

**Review by TT-billteng — APPROVED** [review]

File: pytest.ini (hunk 1 above)
Comment (AleksKnezevic) [reply pytest.ini]: Changed

**Comment by AleksKnezevic** [comment]
> pytest.ini
> 
> Also, can you find a way to not use `SLOW_MATMULS` ?

What do you want me to change about it? I wanted a quick way to disable these MMs for certain pipelines to retain CI coverage while the underlying issue is resolved. I can add more infra to do it via pytest but I'd have to pipe it through all the tests, so it would be a lot of changes for throwaway code.

**Pull request merged**